docs/how-to-guides/enable-streaming-mode.md

```diff
@@ -236,8 +236,8 @@ try:
 
     content_type = response.headers.get('Content-Type')
     if "text/event-stream" in content_type:
-        event_stream = EventStream(response.iter_lines())
-        for event in event_stream:
+        client = SSEClient(response)
```

Comment:
We also need to remove the code of `EventStream` in this PR

Reply:
Understood, Zhidong. I'll remove the EventStream code from the PR as suggested.